### ISSUE TITLE
Test snapshot packages for v0.12.0

### ISSUE THREAD

**Issue by amarthadan** [api3dao/airnode#1839]
Prior to the v0.12.0 release, the snapshot packages and corresponding Docker images need to be tested.

NPM packages:
```
@api3/airnode-abi@snapshot-v0.12.0
@api3/airnode-adapter@snapshot-v0.12.0
@api3/airnode-admin@snapshot-v0.12.0
@api3/airnode-deployer@snapshot-v0.12.0
@api3/airnode-examples@snapshot-v0.12.0
@api3/airnode-node@snapshot-v0.12.0
@api3/airnode-operation@snapshot-v0.12.0
@api3/airnode-protocol@snapshot-v0.12.0
@api3/airnode-utilities@snapshot-v0.12.0
@api3/airnode-validator@snapshot-v0.12.0
```

Docker images:
```
api3/airnode-deployer-dev:snapshot-v0.12.0
api3/airnode-client-dev:snapshot-v0.12.0
api3/airnode-admin-dev:snapshot-v0.12.0
```

**Comment by dcroote** [comment]
## v0.12.0 snapshot testing

tl;dr - everything worked.

I modified `config.json` as follows:

1. Deleted `chains[0].contracts` object in order to test defaulting to API3 deployed AirnodeRrpV0 contract. It worked given the requests below.

2. Deleted `fulfillmentGasLimit` in order to test AirnodeRrpDryRunV0. It worked:

    ```
    INFO	[2023-07-25 05:29:30.239] INFO Gas limit is set to 89152 (AirnodeRrp: 38612 + Fulfillment Call: 50540) for Request
    ```

**A checked box indicates success.**

Test that npm packages `admin`, `deployer`, and `validator`, which have a `--help` CLI argument, are compiled correctly using `npx`

- [x] admin
  - [x] `generate-mnemonic`
- [x] deployer (tested more below)
- [x] validator
  - [x] Succeeds with `--config "packages/airnode-validator/test/fixtures/config.valid.json" --secrets "packages/airnode-validator/test/fixtures/secrets.valid.env"`

The following use the `coingecko-http-gateways` integration in `airnode-examples` on Ethereum sepolia.

Test AWS deployment with `deployer` Docker image:

- Requests
  - [x] Testnet request
  - [x] HTTP gateway request
  - [x] Signed HTTP gateway request
  - [x] Withdrawal request
- Deployer commands (with Docker image)
  - [x] `list`
  - [x] `info`
  - [x] `remove`

## Failures

None.

**Comment by bbenligiray** [comment]
(I'm aware that I said I would test https://github.com/api3dao/airnode/issues/1738 and https://github.com/api3dao/airnode/issues/1673 and still haven't)
